releases/7.6.0.md

```diff
@@ -100,21 +101,23 @@ The minimum supported version of Python is now 3.10. Python 3.11 is encouraged,
         ...
         "license": "AGPL-3.0-only",
         "scripts": {
-            "build_development": "yarn eslint:check && yarn typescript:check && ./media/node_modules/.bin/cross-env NODE_PATH=./media/node_modules NODE_OPTIONS=--max-old-space-size=2048 ./media/node_modules/.bin/webpack --config webpack/webpack.config.dev.js",
-            "build_production": "yarn eslint:check && yarn typescript:check && ./media/node_modules/.bin/cross-env NODE_PATH=./media/node_modules NODE_OPTIONS=--max-old-space-size=2048 NODE_ENV=production ./media/node_modules/.bin/webpack --config webpack/webpack.config.prod.js",
-            "build_test": "yarn eslint:check && yarn typescript:check && ./media/node_modules/.bin/cross-env NODE_PATH=./media/node_modules NODE_OPTIONS=--max-old-space-size=2048 ./media/node_modules/.bin/webpack --config webpack/webpack.config.dev.js --env test=true",
+            "build_development": "yarn eslint:check && yarn ts:check && ./media/node_modules/.bin/cross-env NODE_PATH=./media/node_modules NODE_OPTIONS=--max-old-space-size=2048 ./media/node_modules/.bin/webpack --config webpack/webpack.config.dev.js",
+            "build_production": "yarn eslint:check && yarn ts:check && ./media/node_modules/.bin/cross-env NODE_PATH=./media/node_modules NODE_OPTIONS=--max-old-space-size=2048 NODE_ENV=production ./media/node_modules/.bin/webpack --config webpack/webpack.config.prod.js",
+            "build_test": "yarn eslint:check && yarn ts:check && ./media/node_modules/.bin/cross-env NODE_PATH=./media/node_modules NODE_OPTIONS=--max-old-space-size=2048 ./media/node_modules/.bin/webpack --config webpack/webpack.config.dev.js --env test=true",
             "eslint:check": "./media/node_modules/.bin/eslint ./src --resolve-plugins-relative-to ./media --ext .vue,.ts --parser ./media/node_modules/vue-eslint-parser/index.js",
             "eslint:watch": "./media/node_modules/.bin/nodemon --watch ./src --ext ts,vue --exec yarn --silent eslint:check",
             "gettext:extract": "./media/node_modules/.bin/vue-gettext-extract",
             "gettext:compile": "./media/node_modules/.bin/vue-gettext-compile",
-            "typescript:check": "./media/node_modules/.bin/vue-tsc --noEmit",
-            "typescript:watch": "./media/node_modules/.bin/vue-tsc --watch --noEmit",
+            "ts:check": "./media/node_modules/.bin/vue-tsc --noEmit",
+            "ts:watch": "./media/node_modules/.bin/vue-tsc --watch --noEmit",
             "start": "./media/node_modules/.bin/cross-env NODE_PATH=./media/node_modules NODE_OPTIONS=--max-old-space-size=2048 ./media/node_modules/.bin/webpack serve --config webpack/webpack.config.dev.js"
         },
         ...
     }
     ```
 
+4. In settings.py, remove `"compressor"` from `INSTALLED_APPS`.
```

Comment:
nit: later numbering is off

Reply:
I kind of like letting markdown just auto-number, but since this file already uses incrementing numbers, I'm happy to  keep the file consistent 👍